src/fan.ml

```diff
@@ -32,13 +32,20 @@ let equal t t' =
 
 let log2 a = log a /. log 2.
 
-let v ~hash_size ~entry_size n =
+let v ~hash_size ~entry_size n old_fan_nb =
   let entry_sizef = float_of_int entry_size in
   let entries_per_page = 4096. /. entry_sizef in
   let raw_nb_fans = float_of_int n /. entries_per_page in
   let size = max 0 (int_of_float (ceil (log2 raw_nb_fans))) in
   let nb_fans = 1 lsl size in
   let shift = hash_size - size in
+  let () =
+    match old_fan_nb with
+    | Some on when nb_fans > on ->
+        Log.info (fun m ->
+            m "number of fans has double (from %d to %d)" on nb_fans)
```

Comment:
```suggestion
            m "the number of fan-out entries has changed: %d to %d" on nb_fans)
```

It's technically possible for it to change by any power of two (increasing by more than double if the log size is increased between runs, shrinking if `Index.filter` is used to drop elements).